README.md

```diff
@@ -1,11 +1,24 @@
 credit-my-cc
 ============
 
-Simple webpage for producing a complaint for images on Wikimedia Commons which are improperly reused (in Swedish).
+## Description
+Credit-my-cc is a simple webpage (in Swedish) for producing a complaint when your images on Wikimedia Commons are improperly reused.
 
-Requires [jquery.loadTemplate-1.4.5.min.js](https://github.com/codepb/jquery-template).
+## How to use
+First, add the filename of the image on Wikimedia Commons (without File:) and press Kolla!. Then add the URL where the image has been improperly used in the field **Användning**. If you want to be attributed in another way than what the tool could figure out from the data on Wikimedia Commons, edit the field **Erkännande**. The four radio buttons in the bottom are used to set the tone of the language in the template that is generated on a scale from ☺️ to 🐵. Then press **Skriv!** to generate the template. Copy the text into your email client and send. Hopefully you will get a proper response. Good luck!
```

Comment:
Also works with the prefix, or with the whole url

Reply:
Probably easier to read if broken down into a list

Reply:
Ah, if it works with and without prefix, or with url I believe it would probably be easiest to drop my parenthesis. I agree that a list might be clearer. I'll rearrange that paragraph.